docs/geedocs/docsrc/answer/fusionTutorial/displayRulesPoint.rst

```diff
@@ -275,9 +275,12 @@ Configuring Display Rules for Point Data
          accessible from *myserver.mydomainname*.com/CA_POIs-v001.
 
          To learn more about the options available in the Publish
-         dialog, see :doc:`../geeServerAdmin/createSearchTabs`,
-         :doc:`Snippet profiles <../geeServerAdmin/manageSnippetProfiles>`, and
-         :doc:`WMS <../geeServerAdmin/makeWMSRequests>`.
+         dialog, see 
+
+         - :doc:`Default database <../geeServerAdmin/publishDefaultDatabase>`
+         - :doc:`../geeServerAdmin/createSearchTabs`
```

Comment:
This is not something you changed, but it seems like this line should include a description before the link, like the others. I haven't looked at the page after building it, though, so it might be OK.

Reply:
@tst-lsavoie it was already like that. If there's no description, I believe it automatically uses the page title instead. So I assumed it was left this way on purpose.

Reply:
Confirmed. Once built, this has "Create Search Tabs" as its text. It looks fine once it's built, but we could go ahead and add it here just to be consistent.

Reply:
I went ahead and set the description to be "Search tabs".